ethereum/contracts/BasicInboundChannel.sol

```diff
@@ -2,77 +2,62 @@
 pragma solidity ^0.8.5;
 pragma experimental ABIEncoderV2;
 
-import "./ParachainLightClient.sol";
-import "./BeefyLightClient.sol";
-import "./SimplifiedMMRVerification.sol";
+import "./ParachainClient.sol";
 
 contract BasicInboundChannel {
     uint256 public constant MAX_GAS_PER_MESSAGE = 100000;
     uint256 public constant GAS_BUFFER = 60000;
 
     uint64 public nonce;
 
-    BeefyLightClient public beefyLightClient;
+    ParachainClient public parachainClient;
+
+    struct MessageBundle {
+        uint64 nonce;
+        Message[] messages;
+    }
 
     struct Message {
+        uint64 id;
```

Comment:
These id's are not really used anywhere right, just for events/notification? I guess that's fine, just checking?

Reply:
yeah, its purely so that off-chain clients can track whether messages were processed successfully.